homeassistant/components/tradfri/cover.py

```diff
@@ -5,6 +5,12 @@
 from .base_class import TradfriBaseDevice
 from .const import CONF_GATEWAY_ID, KEY_API, KEY_GATEWAY
 
+ATTRIBUTE_ALIAS = {
+    "battery_level": None,
+    "firmware_version": None,
+    "model_number": "model",
```

Comment:
Model number and firmware version should not be state attributes but part of device_info property, which it already is.

Reply:
Can you please explain how one can get this information out of device_info from a template?

Reply:
I don't believe that's possible. How are you planning to use the information in the template?

Reply:
I explained this in the opening post "but I literally want an automation to close all of my block-out blinds at night but not bother with my see-through ones"

cover.close_cover allows multiple entities to be specified in a single string with entities separated by a cover.
... so with that in mind I you can create an automation that uses the template for entities:
```jinja
{%- set entities = namespace() %}
{%- set entities.entities = '' %}
{%- for state in states.cover -%}
  {%- if 'FYRTUR' in state.attributes.model|upper %}
  {%- if entities.entities != '' %}{% set entities.entities = entities.entities + ',' %}{%- endif %}
  {%- set entities.entities = entities.entities + state.entity_id %}
  {%- endif -%}
{%- endfor %}
{{- entities.entities }}
```

Reply:
Ok. Model would be ok to add to state attributes. We don't need firmware then.